src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSource.cs

```diff
@@ -0,0 +1,140 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+
+
+namespace System.Diagnostics.Tracing
+{
+    /// <summary>
+    /// RuntimeEventSource is an EventSource that represents events emitted by the managed runtime.
+    /// </summary>
+    [EventSource(Guid="5E5BB766-BBFC-5662-0548-1D44FAD9BB55", Name = "System.Runtime")]
```

Comment:
Is this the right name? Just "System.Runtime"?

Reply:
Where does the GUID: `5E5BB766-BBFC-5662-0548-1D44FAD9BB55` come from? Was it obtained from something like this:
```csharp
TraceEventProviders.GetEventSourceGuidFromName("System.Runtime");
```